global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "2.1.300"
+    "dotnet": "3.0.100-preview4-010597"
```

Comment:
Does the preview3 build `3.0.100-preview3-010431` have this support?

To be sure that we don't break the source-build N-1 chain ([description of N-1](https://github.com/dotnet/source-build/blob/master/Documentation/EngineeringPlanLinuxDistroRepos.md#n-1-toolset)), we should stick to releases.

/cc @dseefeld @crummel @dleeapho

Reply:
That is a good question. I don't know, let me see what I can find out.

Reply:
Oh, and this is where the actual value is, since Core-Setup isn't on Arcade SDK yet:

https://github.com/dotnet/core-setup/blob/e9692c95240cf23da279fb3f69942591674e6d44/DotnetCLIVersion.txt#L1

I think the global.json is only here to prevent Arcade dependency flow from failing.